src/kafka-util/src/client.rs

```diff
@@ -233,6 +247,10 @@ impl ClientContext for MzClientContext {
         }
     }
 
+    fn stats(&self, statistics: Statistics) {
```

Comment:
how often does rdkafka call this?

Reply:
It's controlled by the `statistics.interval.ms` config option on the client